docs/docs/articles/istio.md

```diff
@@ -0,0 +1,120 @@
+# Using Istio
```

Comment:
I'd change the structure a bit.

You can start with a TL;DR; on Istio but that is completely optional.

I'd first start with introducing the problem statement with Testkube & Istio and then introduce a solution, currently it is hard to immediatelly get context what is the problem statement.

Context - optional

Problem Stamenent

Solution

Examples

Testkube compatibility (mention what is officially supported and what not)

Reply:
The problems are stated below, and the context is required to understand them. These are not Testkube-specific problems.

I think this fine, unless you have some concrete suggestions.